docs/docs/api/09-Remote-Device-Controller.md

```diff
@@ -0,0 +1,114 @@
+# Remote Device Controller
+
+:::caution
+
+The remote device controller is currently in __preview__ release and only works on __Windows__ and for __Android__ devices controlled from a __Windows__ machine.
+
+The installer only works on __Windows__.
+
```

Comment:
There are basically two device controllers at the moment. The legacy one that still provides support for MacOS and Linux (X11) and the new one described here. Shouldn't we differentiate this?

Reply:
Maybe I do not get the full picture here:

The Remote Device Controller for me is the one that only works on Windows and is described on this page. But it will eventually be available for Linux and macOS too.

And then there is the UiController.

Is that correct? If so we already have a separate page for the UiController config and we do not reference the Remote Device Controller anywhere in our docs. The confusion should be minimal here in my opinion.